lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -147,9 +147,8 @@
             accountsSectionData = [
                 {
                     title: gettext('Linked Accounts'),
-                    subtitle: gettext(
-                        'You can link your social media accounts to your edX account to make signing in to edx.org ' +
-                        'and the edX mobile apps easier.'
+                    subtitle: window.interpolate_text(
+                        gettext('You can link your social media accounts to your {platform_name} account to make signing in to {platform_name} and the {platform_name} mobile apps easier.'), {platform_name: platformName}     /* jshint ignore:line */
```

Comment:
Again I'd remove the duplication of ``{platform_name}`` here. @srpearce, what would you suggest? Maybe:

    You can link your social media accounts to simplify signing in to {platform_name}.